core/cleaner/src/main/java/com/cognifide/aet/cleaner/CleanerScheduler.java

```diff
@@ -156,11 +118,10 @@ private String registerCleaningJob() throws SchedulerException {
 
     final ImmutableMap<String, Object> jobData = ImmutableMap.<String, Object>builder()
         .put(CleanerJob.KEY_ROUTE_BUILDER, metadataCleanerRouteBuilder)
-        .put(CleanerJob.KEY_KEEP_N_VERSIONS, keepNVersions)
-        .put(CleanerJob.KEY_REMOVE_OLDER_THAN, removeOlderThan)
-        .put(CleanerJob.KEY_COMPANY_FILTER, companyName)
-        .put(CleanerJob.KEY_PROJECT_FILTER, projectName)
-        .put(CleanerJob.KEY_DRY_RUN, dryRun)
+        .put(CleanerJob.KEY_REMOVE_OLDER_THAN, config.removeOlderThan())
```

Comment:
Please add .put(CleanerJob.KEY_KEEP_N_VERSIONS, config.keepNVersions()). We don't have equivalent for .put(CleanerJob.KEY_KEEP_N_VERSIONS, keepNVersions)

Reply:
Done